### PR TITLE
Bump serverless version to v1.71.1 (#1)

### PR DESCRIPTION
**Pull request by jufemaiz**
Latest release as per <https://www.npmjs.com/package/serverless>

### CODE REVIEW

**Comment by razbomi** [comment]
Yo @jufemaiz,

 Serverless is now at 1.70.1
https://github.com/serverless/serverless/releases

Any change you can bump this before you commit gets merged?
Also may I suggest pinging or assign @flemay as the reviewer, if not already done.

**Comment by jufemaiz** [comment]
* Update as @razbomi requested.
* Ping @flemay for asigning.

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-SERVERLESS_VERSION = 1.67.2
+SERVERLESS_VERSION = 1.68.0
```
Comment (razbomi) [inline Makefile:1]: 1.70.1 in the makefile as well... Seems like there is codes duplication in Make and Docker files.

**Comment by razbomi** [comment]
Hi @flemay or @yunspace 

What is the best way to get this merged? Thanks you.

**Comment by jufemaiz** [comment]
Bumped to latest ([v1.71.1](https://github.com/serverless/serverless/releases/tag/v1.71.1))

**Comment by flemay** [comment]
Sorry I missed this one! Will review it today or tomorrow! Thank you! :D

File: Makefile [hunk 2]
```diff
@@ -1,9 +1,9 @@
-SERVERLESS_VERSION = 1.67.2
+SERVERLESS_VERSION = 1.71.1
 IMAGE_NAME ?= amaysim/serverless:$(SERVERLESS_VERSION)
 TAG = $(SERVERLESS_VERSION)
 
 build:
-	docker build -t $(IMAGE_NAME) .
+	docker build --build-arg SERVERLESS_VERSION=$(SERVERLESS_VERSION) -t $(IMAGE_NAME) .
```
Comment (flemay) [inline Makefile:6]: I <3 this!

**Comment by flemay** [comment]
Merged and built! Thank you all for your contributions! 👍

**Comment by jufemaiz** [comment]
Ironically it's already bumped up to v1.71.1 ><

**Comment by flemay** [comment]
@jufemaiz @razbomi Just in case you didn't know, we've now automated the build process and the image gets built **weekly** with the latest Serverless version. So it should not require any human action from now on! More details in this [PR](https://github.com/amaysim-au/docker-serverless/pull/33).

**Pull request merged**